src/rt/arc.rs

```diff
@@ -45,6 +45,10 @@ pub(super) enum Action {
 impl Arc {
     pub(crate) fn new() -> Arc {
         rt::execution(|execution| {
+            if execution.log {
+                println!("Arc::new");
```

Comment:
Perhaps we should include something identifying the Arc (maybe the internal loom object ID or a memory address?) in this message and the other messages in this module? 

Although those don't convey a whole lot of semantic information about which `Arc` in the program the messages refer to, they do allow matching the `Arc::new`, Arc::ref_inc`, `Arc::ref_dec` messages and so on. Paired with logging in the code under test, users _could_ probably track different `Arc`s in their code based on an identifier added by loom.

Reply:
Done.